dockstore-webservice/src/main/resources/swagger-templates/api.mustache

```diff
@@ -41,7 +41,7 @@ import jakarta.validation.constraints.*;
 {{#hasConsumes}}@Consumes({ {{#consumes}}"{{{mediaType}}}"{{#hasMore}}, {{/hasMore}}{{/consumes}} }){{/hasConsumes}}
 {{#hasProduces}}@Produces({ {{#produces}}"{{{mediaType}}}"{{#hasMore}}, {{/hasMore}}{{/produces}} }){{/hasProduces}}
 @io.swagger.annotations.Api(description = "the {{{baseName}}} API")
-@io.swagger.v3.oas.annotations.tags.Tag(name = "GA4GH", description = ResourceConstants.GA4GH)
```

Comment:
Decided to specially indicate that this is for the beta api since it wasn't properly hooked up in the first place